src/Components/Navbar.js

```diff
@@ -1,37 +1,37 @@
-import React  from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import React from 'react';
 import AppBar from '@material-ui/core/AppBar';
 import Tab from '@material-ui/core/Tab';
 import TabContext from '@material-ui/lab/TabContext';
 import TabList from '@material-ui/lab/TabList';
+import styled from 'styled-components';
 
-const useStyles = makeStyles((theme) => ({
-    root: {
-        flexGrow: 1,
-        backgroundColor: theme.palette.background.paper,
-    },
-}));
+const StyledAppBar = styled(AppBar)`
+        background : #222;
+    `;
```

Comment:
I think adding ...Styled to end is better that adding it to the beginning. so,
`AppBarStyled` is a better name.